### PR TITLE
chore: release fxcorevisor v7.2.0-rc2

### PR DESCRIPTION
**Pull request by nulnut**
# Description

Closes: #XXXX

merge after fxcore docker image build

@fx0x55 @zakir-code

---

## Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

* [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
* [ ] confirmed `!` in the type prefix if API or client breaking change
* [ ] targeted the correct branch (see [PR Targeting](https://github.com/cosmos/cosmos-sdk/blob/main/CONTRIBUTING.md#pr-targeting))
* [ ] provided a link to the relevant issue or specification
* [ ] reviewed "Files changed" and left comments if necessary
* [ ] included the necessary unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/main/CONTRIBUTING.md#testing)
* [ ] added a changelog entry to `CHANGELOG.md`
* [ ] updated the relevant documentation or specification, including comments for [documenting Go code](https://blog.golang.org/godoc)
* [ ] confirmed all CI checks have passed

## Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

* [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
* [ ] confirmed all author checklist items have been addressed
* [ ] reviewed state machine logic, API design and naming, documentation is accurate, tests and test coverage

### CODE REVIEW

File: tools/cosmovisor/upgrades-info/v7.2.x/fxcore/upgrades/v7.2.x/upgrade-info.json
Comment (zakir-code) [inline tools/cosmovisor/upgrades-info/v7.2.x/fxcore/upgrades/v7.2.x/upgrade-info.json:1]: 这里一定要填吗， 主网是没有这个升级的

Comment (zakir-code) [reply tools/cosmovisor/upgrades-info/v7.2.x/fxcore/upgrades/v7.2.x/upgrade-info.json:1]: 能不能优化一下

Comment (zakir-code) [reply tools/cosmovisor/upgrades-info/v7.2.x/fxcore/upgrades/v7.2.x/upgrade-info.json:1]: 另外，我想把这部份的功能移到fxcore项目

Comment (nulnut) [reply tools/cosmovisor/upgrades-info/v7.2.x/fxcore/upgrades/v7.2.x/upgrade-info.json:1]: > 这里一定要填吗， 主网是没有这个升级的

要填，要根据高度对比判断使用哪一个版本

Comment (nulnut) [reply tools/cosmovisor/upgrades-info/v7.2.x/fxcore/upgrades/v7.2.x/upgrade-info.json:1]: > 另外，我想把这部份的功能移到fxcore项目

这个怎么移动到fxcore？这是控制启动fxcore的项目

**Review by zakir-web3 — APPROVED** [review]

**Pull request merged**